### ISSUE TITLE
[wg/pointerevents] Pointer Events Working Group 2023

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#416]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://w3c.github.io/charter-drafts/2023/pointer-events.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 
 - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2021%2F03%2Fpointer-events-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2023%2Fpointer-events.html), and any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

None. This is a Group with a very narrow scope...

Known or potential areas of concern: 

none.

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

https://github.com/w3c/charter-drafts/issues

Anything else we should think about as we review? 

nope

**Comment by plehegar** [comment]
@patrickhlauke , got any update for the CR and PR milestones for PE3?

**Comment by plehegar** [comment]
Given that this is an existing straightforward Working Group, I'm requesting the advance notice.

**Comment by patrickhlauke** [comment]
@plehegar we're aiming to sort out the last few blockers, with current goal of being ready to go to CR around TPAC 2023 time if that's at all feasible

**Comment by plehegar** [comment]
So Q3 2023 for CR. Does that mean Q4 2023 for PR (ie will we have the implementations?)?

**Comment by patrickhlauke** [comment]
@plehegar hoping so, yes (i *think* we're not too far off with implementations, though need to have a closer look at all the WPTs we're currently updating/creating)

**Comment by himorin** [comment]
No comment nor request from i18n

**Comment by plehegar** [comment]
AC Review: https://lists.w3.org/Archives/Member/w3c-ac-members/2023JulSep/0025.html

**Comment by plehegar** [comment]
@patrickhlauke , how would you like me to update the timeline?

**Comment by himorin** [comment]
Quite minor point, but the latest charter extension is not recorded in the charter history table.

**Comment by svgeesus** [comment]
Charter says: Exclusion Draft: [Pointer Events (Level 3), 12 December 2020](https://www.w3.org/TR/2019/WD-pointerevents3-20191212/)

But the link is to W3C First Public Working Draft 12 December 2019. [History](https://www.w3.org/standards/history/pointerevents3/)

**Comment by svgeesus** [comment]
I agree that Expected completion: Q4 2023 is impossible.

**Comment by plehegar** [comment]
Wide review will take 2 months, starting November 1. We should be in CR in January.

I'll update the charter.

**Comment by plehegar** [comment]
Approved: https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0033.html
